src/cpp/include/openvino/genai/llm_pipeline.hpp

```diff
@@ -105,7 +106,8 @@ class OPENVINO_GENAI_EXPORTS LLMPipeline {
     LLMPipeline(
         const std::string& path, 
         const std::string& device="CPU", 
-        const ov::AnyMap& plugin_config={}
+        const ov::AnyMap& plugin_config={},
+        OptionalSchedulerConfig scheduler_config
```

Comment:
can scheduler be passed as plugin_config option?
Then in CB pipeline, we can extract this option and remaining ones will be passed to compile_model